tools/postReleaseScripts/publishGHRelease.ps1

```diff
@@ -60,17 +70,16 @@ function GetDescriptionFromChangelog
     return $sb.ToString()
 }
 
-$tag = "v$Version"
-
 $releaseParams = @{
     Organization = $TargetFork
     Repository = $Repository
-    Tag = $tag
-    ReleaseName = $tag
-    Branch = "release/$Version"
+    Tag = $ReleaseTag
+    ReleaseName = $ReleaseTag
+    Branch = $BranchName
     AssetPath = $AssetPath
-    Prerelease = [bool]($Version.PreReleaseLabel)
```

Comment:
How come we're expected to pass this in now?

Reply:
Because vscode extension versions don't come with a prerelease label